src/admin/shared/constants/admin-core.routes.ts

```diff
@@ -0,0 +1,6 @@
+import { ROUTE_PARTS } from '@meemoo/admin-core-ui';
+
+export const ADMIN_CORE_ROUTE_PARTS: typeof ROUTE_PARTS = {
+	...ROUTE_PARTS,
+	admin: 'beheer' as 'admin', // TODO
```

Comment:
i wonder if we would get away with making this /admin for avo as well, so both platforms are the same
but then also add a redirect in the avo client to redirect /beheer* => /admin* ?
It might make the admin-core code a bit cleaner?